app/src/main/java/org/jellyfin/androidtv/ui/presentation/MyImageCardView.java

```diff
@@ -230,15 +234,26 @@ public void clearBanner() {
     }
 
     public void setUnwatchedCount(int count) {
+        WatchedIndicatorBehavior showIndicator = get(UserPreferences.class).get(UserPreferences.Companion.getWatchedIndicatorBehavior());
```

Comment:
I'd rather keep this class "dumb" and not lookup settings in here, would be better to add that logic to the calling side.